test/test_cli.py

```diff
@@ -0,0 +1,75 @@
+import shutil
+import unittest
+import subprocess
+import os
+from app import application
+
+
+class TestApplication(unittest.TestCase):
+    def test_initialize_repo(self):
+        try:
+            shutil.rmtree("../bitcointranscripts", ignore_errors=True)
+            subprocess.run(["rm", "-rf", "test"])
+        except Exception as e:
+            self.fail(e)
+
+    def test_find_source_type(self):
+        video = application.find_source_type("B0HW_sJ503Y") == "video"
+
+        playlist = application.find_source_type("PLPQwGV1aLnTuN6kdNWlElfr2tzigB9Nnj") == "playlist"
+
+        audio = application.find_source_type(
+            "https://anchor.fm/s/12fe0620/podcast/play/32260353/https%3A%2F%2Fd3ctxlq1ktw2nl.cloudfront.net%2Fstaging"
+            "%2F2021-3-26%2Fdc6f12e7-a547-d872-6ef6-7acfe755a692.mp3") == "audio"
+        self.assertTrue(audio and playlist and video)
+
+    def test_download_audio_file(self):
+        audio = application.get_audio_file(
+            "https://anchor.fm/s/12fe0620/podcast/play/32260353/https%3A%2F%2Fd3ctxlq1ktw2nl.cloudfront.net%2Fstaging"
```

Comment:
A metric of usefulness for this test suite should be speed. If you are going to test this (which I think is debatable) you should fetch the shortest file possible. On poor internet this took me ~25 seconds. That's way too much.

In general though, I don't think it's very useful to test the fetching of youtube videos or audio files. Maybe that can be a sanity check test but it shouldn't be run often.